proc_master.go

```diff
@@ -426,3 +427,12 @@ func token() string {
 	rand.Read(buff)
 	return hex.EncodeToString(buff)
 }
+
+// On Windows, include the .exe extension, noop otherwise.
+func extension() string {
```

Comment:
It was recommended to place this in an `init` function in `proc_windows_slave.go`, but I think this is more clear. `proc_windows_slave.go` works incidentally, the file itself does not use `tmpBinPath` and is the `_slave`, not `_master` file. If there were a `proc_master_windows.go` I could see a better argument, but even then I feel that this is more explicit about what is happening.

Reply:
Agreed, avoid init where possible 👍